lib/sunbeam-ws.js

```diff
@@ -226,17 +231,20 @@ class MandelbrotEosfinex extends EventEmitter {
     const {
       keys,
       client,
-      scatter
+      ual
```

Comment:
I'm sorry, I might be missing something, are we dropping scatter?

Also, is there a config entry that could be used to setup `ual`?

Reply:
`setAuth` is only used by the UI. Will port the example code in sunbeam to also use this, see comment below.

Reply:
fixed now